### DISCUSSION TITLE
The GSSOC discussion board

### DISCUSSION THREAD

**Discussion by qascade** [qascade/dcr#46] [Q&A]
You can put any questions/queries related to dcr here.

**Comment by SaxenaAnushka102** [comment]
Hello @qascade !! I just came across the project recently on the GSSoC page. I really liked the idea & would love to get involved in the project. As a newbie, what are some resources that I can refer to get acquainted with the tech-stack of the project? Also, please can you suggest a way on how can I clone the project when I have access to only Windows OS?
Thanks,
Anushka Saxena.

**Reply by qascade** [reply]
Hey @SaxenaAnushka102, Thanks for taking interest in this project. For the tech stack, please look at these [guidelines](https://github.com/qascade/dcr/blob/main/GSSoC_GUIDELINES.md). You can fork/clone this repo using git but you won't be able to contribute to this project using Windows, you will have to have some sort of Linux setup on VM or through dual boot. 
For theoretical background, you may look at the references at the end of readme.
